jekyll/_cci2/rotate-project-ssh-keys.adoc

```diff
@@ -0,0 +1,110 @@
+---
+contentTags:
+  platform:
+  - Cloud
+---
+= Rotate project SSH keys
+:page-layout: classic-docs
+:page-liquid:
+:page-description: How to guide detailing how to rotate your project SSH keys.
+:icons: font
+:toc: macro
+:toc-title:
+
+To revoke usage of a project SSH key, rotate the key pairs by following the steps outlined below. There are separate instructions for each supported version control system (VCS).
+
+When using project SSH keys, CircleCI holds the private key, and the target system (typically a VCS, machine or other SSH based system) holds the matching public key via its authorization system. To rotate SSH keys both the private and public keys need to be removed.
+
+[#github-projects]
+== Github projects
+
+Go to **Project Settings > SSH Keys** to view SSH keys set up for your project.
+
+[#rotate-a-deploy-key-github]
+=== Rotate a deploy key
+
+. Take note of the current key information to rotate, including the fingerprint. You can also click the keyname to open the related GitHub page that should list the public key.
+. Delete the deploy key by clicking the **X**.
+. Add a new deploy key by clicking **Add Deploy Key**.
+. Go to GitHub’s repository project settings to delete the matching public key. The github URL is typically `https://github.com/<org>/<project>/settings/keys`, or you may already have the page open if you clicked on the keyname in step 1. The keys are named `CircleCI`. Removing any key titled `CircleCI` created before the rotation is recommended. The new public SSH key will be automatically added once the old key is deleted.
+
+[#rotate-a-user-key-github]
+=== Rotate a user key
+
+If you have set up user keys for your project, follow these steps:
+
+. Take note of the current key information to rotate, including the fingerprint. You can also click the keyname to open the related Github page that should list the public key.
+. Delete the deploy key by clicking the **X**.
+. Add a new user key by clicking **Add User Key**.
+. Go to GitHub’s User settings to delete the matching public key. The github URL is typically `https://github.com/settings/keys`or you may already have the page open if you clicked on the keyname in step 1. Removing any key titled `CircleCI` created before this rotation is recommended. The new public SSH key will be automatically added once the old key is deleted (page may need to be refreshed to view the key).
+
+CAUTION: The user key name contains the project name, however, a user key may give access to more projects within the organization.
+
+NOTE: If using organization SSO, you can take note of which one is currently authorized. For the newly created key, if using organization SSO, and access is needed, you will need to reauthorize it.
```

Comment:
>  which one is currently authorized

Not super clear which "one" this is referring to. User vs project key? I looked back at the Google doc and this would be my guess?

Reply:
'one' in that context is 'organization'. For example I am part of 4 orgs and my key is authorized only for two of them.

Reply:
I will try reworking this note so it's easier to read